pkg/services/ngalert/schedule/schedule.go

```diff
@@ -580,3 +566,20 @@ func (sch *schedule) getRuleExtraLabels(evalCtx *evaluation) map[string]string {
 	}
 	return extraLabels
 }
+
+func ScheduleUserFor(orgID int64) *user.SignedInUser {
```

Comment:
```suggestion
func SchedulerUserFor(orgID int64) *user.SignedInUser {
```
unbelievably tiny nitpick